### DISCUSSION TITLE
What am I doing wrong here?

### DISCUSSION THREAD

**Discussion by jtrwallace** [CodyBerenson/PGMA-Modernized#220] [Q&A]
I am having a lot of trouble getting the newish agents to work. The logs are saying they can't get the token or machine ID from my preferences file. I notice in the path to the preference plist there are two slashes between Library and Preferences. Is that incorrect? Any help is appreciated. 

[com.plexapp.agents.AEBN.log](https://github.com/CodyBerenson/PGMA-Modernized/files/10471953/com.plexapp.agents.AEBN.log)

**Comment by JPH71** [comment]
If you look through all the issues, you will see that if your system can
not get the token automatically that you can paste it in your preferences
manually...

Thanks for sending the log... I will look at it

**Comment by CodyBerenson** [comment]
@jtrwallace 
Hi Joshua, here is a link to the token discussion Jason refers to above.  Hope it helps.

https://github.com/CodyBerenson/PGMA-Modernized/issues/213#issuecomment-1374909436

**Comment by JPH71** [comment]
Have you managed to get this to work???
